controllers/content/content.controller.js

```diff
@@ -47,70 +46,62 @@ const getContent = async (req, res) => {
       filterValue,
     });
     const sortClause = {
-      [sortType]: 1
+      [sortType]: 1,
     };
     if (sortDirection === CONTENT.SORT_DIRECTION.DESC) {
       sortClause[sortType] = -1;
     }
     const skip = paginationUtil.getOffset();
     const limit = paginationUtil.getLimit();
 
-    const contentData = await ContentModel.find(findQuery)
-      .sort(sortClause)
-      .skip(skip)
-      .limit(limit);
+    const contentData = await ContentModel.find(findQuery).sort(sortClause).skip(skip).limit(limit);
 
     const responseObject = {
       contentData,
       pageNumber,
       pageSize,
-    }
+    };
     return successResponse({
       res,
       responseObject,
-    })
+    });
   } catch (error) {
     return errorResponse({
       res,
       error,
       responseMessage: 'Error listing content.',
-    })
+    });
   }
-}
+};
 
+// Create a new Content Post
 const postContent = async (req, res) => {
   try {
-    const {
-      title,
-      content,
-      name,
-      email,
-      otp
-    } = req.body;
+    const { title, content, name, email, otp } = req.body;
 
     const newPostData = {
       title,
-      content, 
+      content,
       type: CONTENT.TYPES.DEFAULT,
       flag: CONTENT.FLAGS.PENDING,
       isSpam: CONTENT.SPAM_TYPES.DEFAULT,
       tags: [],
       reviews: {
         isReviewed: false,
-        reviewers: []
+        reviewers: [],
       },
       sourceMetaData: {},
       acl: {
```

Comment:
can skip it here. acl will get added during verification phase as it needs user id as well. (User gets created during the verification phase)